### PR TITLE
[BXMSDOC-2481] Marked blocks in templates

### PR DESCRIPTION
**Pull request by mramendi**
In order to enable the user to modify templates easily, marked out blocks using comments. The work seems complete except for the immutable monitoring template, which I won't do right now.

### CODE REVIEW

File: templates/rhpam70-kieserver-mysql.yaml [hunk 1]
```diff
@@ -456,22 +460,21 @@ objects:
...
-            value: "${KIE_SERVER_MYSQL_DB}"
           - name: RHPAM_JNDI
             value: "${KIE_SERVER_PERSISTENCE_DS}"
-          - name: RHPAM_DRIVER
-            value: "mysql"
-          - name: RHPAM_JTA
-            value: "true"
           - name: RHPAM_TX_ISOLATION
             value: "TRANSACTION_READ_COMMITTED"
+## PostgreSQL driver settings BEGIN
```
Comment (sutaakar) [inline templates/rhpam70-kieserver-mysql.yaml]: This should be MySQL, not PostgreSQL

File: templates/rhpam70-kieserver-mysql.yaml [hunk 2]
```diff
@@ -482,6 +485,9 @@ objects:
             value: "3306"
           - name: TIMER_SERVICE_DATA_STORE
             value: "${APPLICATION_NAME}-mysql"
+## PostgreSQL driver settings END
```
Comment (sutaakar) [inline templates/rhpam70-kieserver-mysql.yaml]: Same as above

File: templates/rhpam70-prod.yaml [hunk 3]
```diff
@@ -1285,6 +1311,8 @@ objects:
         - name: "${APPLICATION_NAME}-postgresql-pvol"
           persistentVolumeClaim:
             claimName: "${APPLICATION_NAME}-postgresql-claim-2"
+## PostgreSQL deployment config 2 BEGIN
```
Comment (sutaakar) [inline templates/rhpam70-prod.yaml]: I suppose it should be END here

**Comment by mramendi** [comment]
I have fixed the errors that Karel found and renamed the new squashed commit. Thanks!

**Comment by jiripetrlik** [comment]
@sutaakar @mramendi Is this PR finished? We need to have final version of templates ASAP.

**Pull request merged**